api-contracts/openapi/paths/monitoring/monitoring.yaml

```diff
@@ -0,0 +1,41 @@
+probe:
+  post:
+    x-resources: ["tenant"]
+    description: Triggers a workflow to check the status of the instance
+    summary: Detailed Health Probe For the Instance
+    operationId: monitoring:post:runProbe
+    parameters:
+      - description: The tenant id
+        in: path
+        name: tenant
+        required: true
+        schema:
+          type: string
+          format: uuid
+          minLength: 36
+          maxLength: 36
+    responses:
+      "200":
+        content:
+          application/json:
+            schema:
+              $ref: "../../components/schemas/_index.yaml#/MonitoringRunProbe"
```

Comment:
See above, I don't think this endpoint needs to return an object

Reply:
ok, didn't realize I could just return a nil from the handler and it would be fine. Removed this object